### DISCUSSION TITLE
Add content-100 ... 300, like base

### DISCUSSION THREAD

**Discussion by silveltman** [saadeghi/daisyui#1781] [Ideas / feature requests]
Base colors has 3 values: 100, 200 and 300.

It would be gret to also have this for text/content. For example:
base-content-100
base-content-200
base-content-300

This would allow us to apply a different color to a heading than to body text, which is something I personally always do. It bring visual hierarchy.

**Comment by saadeghi** [comment]
I think applying the opacity would give you the same result. Like `text-base-content/50` which would be `base-content` color when 50% opacity.


I'm working on daisyUI v3 right now and trying to provide a solution to change the lightness of any color using utility classes. If it goes well, it will be a huge improvement.

Let me know if you have any questions.

**Comment by silveltman** [comment]
Sounds like a solution, for now I will use the unstyled version. 

Any idea when V3 will launch? And can you share some more details on the improvements? :)
